docroot/sites/all/modules/features/forms/forms.module

```diff
@@ -384,6 +384,50 @@ function _process_collective_bargaining_videos(&$entityform) {
 }
 
 /**
+ * Process eMediation Competition registration form (Scheinman)
+ * If there's a second participant, send her an email, too.
+ */
+function _process_emediation_competition_reg(&$entityform) {
+  // Regular email
+  $admin_email = _forms_get_admin_email($entityform);
+  _forms_send_emails($entityform, $admin_email);
+
+  // Email to second team member if it's a two-person team
+  if (sizeof($entityform->field_email_additional['und'])) {
```

Comment:
sizeof is an alias for count in php, and rarely used. Did you consider using !empty?